cmd/acra-server/acra-server.go

```diff
@@ -106,12 +106,12 @@ func main() {
 	enableHTTPAPI := flag.Bool("http_api_enable", false, "Enable HTTP API")
 
 	useTLS := flag.Bool("acraconnector_tls_transport_enable", false, "Use tls to encrypt transport between AcraServer and AcraConnector/client")
-	tlsKey := flag.String("tls_key", "", "Path to private key that will be used in TLS handshake with AcraConnector as server's key and Postgresql as client's key")
+	tlsKey := flag.String("tls_key", "", "Path to private key that will be used in TLS handshake with AcraServer as server's key and database as client's key")
```

Comment:
between AcraServer and database, right?

Reply:
client/acra-connector <-> acra-server and acra-server <->database. So leave `with AcraConnector` as it was before : )

Reply:
ah, got it!